src/main/java/appeng/helpers/iface/PatternProviderLogic.java

```diff
@@ -388,12 +389,14 @@ public GenericStack getUnlockStack() {
     private Set<Direction> getActiveSides() {
         var sides = host.getTargets();
 
-        // Skip sides with grid connections to other pattern providers
+        // Skip sides with grid connections to other pattern providers and to interfaces connected to the same network
         var node = mainNode.getNode();
         if (node != null) {
             for (var entry : node.getInWorldConnections().entrySet()) {
                 var otherNode = entry.getValue().getOtherSide(node);
-                if (otherNode.getOwner() instanceof PatternProviderLogicHost) {
+                if (otherNode.getOwner() instanceof PatternProviderLogicHost
+                        || (otherNode.getOwner() instanceof InterfaceLogicHost
+                                && otherNode.getGrid().equals(mainNode.getGrid()))) {
```

Comment:
Weird that this check didn't exist for pattern providers? Why shouldn't you be able to push to a pattern provider on a different network?

Reply:
I can't see myself why not, but I wasn't sure if there was any specific intent to disallowing any and all pattern providers from receiving other providers' inputs.

Reply:
Hmmm this might have been an intended recent change. @shartte does that ring a bell?